src/configuration.js

```diff
@@ -179,15 +184,23 @@ export default class Configuration {
             );
         }
 
+        let upsamplingEnabled = false;
+        if (config.upsampling && config.upsampling.enabled !== null && config.upsampling.enabled) {
```

Comment:
do we need the null check? according to https://developer.mozilla.org/en-US/docs/Glossary/Falsy any uninitialized value should make this false